mpl_interactions/controller.py

```diff
@@ -271,15 +297,40 @@ def _ipython_display_(self):
 
 
 def gogogo_controls(
-    kwargs, controls, display_controls, slider_formats, play_buttons, allow_dupes=False
+    kwargs,
+    controls,
+    display_controls,
+    slider_formats,
+    play_buttons,
+    extra_controls=None,
+    allow_dupes=False,
 ):
-    if controls:
+    if controls or (extra_controls and not all([e is None for e in extra_controls])):
+        if extra_controls is not None:
+            controls = [controls] + extra_controls
+
         if isinstance(controls, tuple):
             # it was indexed by the user when passed in
             extra_keys = controls[1]
             controls = controls[0]
             controls.add_kwargs(kwargs, slider_formats, play_buttons, allow_duplicates=allow_dupes)
             params = {k: controls.params[k] for k in list(kwargs.keys()) + list(extra_keys)}
+        elif isinstance(controls, list):
```

Comment:
this probs breaks if controls was indexed when passed in and there are extra controls

Reply:
i think this is fixed now. what a nightmare

Reply:
oh no. i lost it in a rebase :'(

that was so much work :'(

i really goofed

Reply:
much less hassle to fix the second go around! thank goodness